scipy/stats/__init__.py

```diff
@@ -265,6 +265,7 @@
    brunnermunzel
    combine_pvalues
    jarque_bera
+   pagel
```

Comment:
Immediate first thought: `pagel` like "bagel"? This doesn't seem like a good function name. Maybe `page_l`?

Reply:
Thanks for taking a look. `page_l` was the original name, but I changed it for consistency with `kendalltau`, `spearmanr`, `pearsonr`, `johnsonsb`, `johnsonsu`, `mannwhitneyu`, and `friedmanchisquare`: surname adjoined with variable name. There are some functions that have a `_` after the surname, but they don't have the name of a variable after them; they're more of a description (e.g. `fisher_exact`, `yeojohnson_normmax`.

Reply:
@rgommers, I'd like to merge this soon.  Are you OK with `pagel`?

Reply:
Sure. It's a terrible name, but at least it's consistent with the other terrible names - and `page_l` also won't tell anyone what the function does.

Reply:
I've found two versions in R, one called `page.trend.test` (in the `crank` package) and another called `page.test` (in the `cultevo` package).  In [this blog post](https://kevinstadler.github.io/cultevo/articles/page.test.html), the author of the `cultevo` package argues that the test isn't actually a *trend* test. If that argument is convincing, then including `trend` in the name isn't quite accurate.

Perhaps something with the word `ordered`, e.g. `page_ordered_test` (with or without the underscores, with or without the word `test`).   The name Page is clearly associated with this test, so I think we want to keep `page` in there.

Naming things is hard.  Suggestions for names that are not terrible would be appreciated!

Reply:
`page_test` or `page_l_test` is what I'd choose. We have `ttest`, `binomtest` etc. as well - makes it a bit more descriptive.

Reply:
The only thing preventing me from merging this is @rgommers objection to the name.  I know @mdhaber prefers `pagel`, and (for better or worse), that style is consistent with many other names in `scipy.stats`.  I don't have a strong preference, and when that happens I try to go with the original author's preference.  Does anyone else have input?  Is there a set of objective criteria we can apply to evaluate the quality of a name?